pypxe-server.py

```diff
@@ -85,9 +89,22 @@
             dargs.update(loadedcfg)
             args = argparse.Namespace(**dargs)
 
+        # setup logger
+        logger = logging.getLogger(sys.argv[0])
```

Comment:
If running as such from the command line `$ python pypxe-server.py` then wouldn't `sys.argv[0]` simply be the string "pypxe-server.py"? If so then that's fine but we should confirm that if this is run twice that the second time it doesn't overwrite the original log. Can we either make a new log or add onto the previous log?